pyserializer/fields.py

```diff
@@ -58,11 +60,24 @@ def __init__(self,
         :param validators: List of validators that should be ran when
             deserializing the field. This list will be appended along with
             the default_validators defined on each field.
+        :param error_messages: (dict) Custom error message on the field.
         """
         self.source = source
         self.label = label
         self.help_text = help_text
         self.validators = self.default_validators + (validators or [])
+        self.error_dict = OrderedDict()
+        if error_messages:
+            self.error_dict['type_name'] = kwargs.pop(
+                'type_name',
+                self.type_name
+            )
+            self.error_dict['type_label'] = kwargs.pop(
+                'type_label',
+                self.type_label
+            )
+            self.error_dict['message'] = error_messages['invalid']
```

Comment:
For your example I see a couple of options for the API that would allow you to customize the error message for each validator:

```python
# Option 1
# Override the error messages from the validators at the field level
class UserDeserializer(Serializer):
  age = fields.IntegerField(
    validators=[MinValueValidator(18), MaxValueValidator(30)],
    error_messages={
      'min_value': 'You must be at least 18 years old to signup',
      'max_value': 'You are too old...',
    },
  )

# Option 2
# Customize the error messages at the validator level
class UserDeserializer(Serializer):
  # Custom error messages per validator
  age_1 = fields.IntegerField(
    validators=[
        MinValueValidator(18, error_message='You must be at least 18 years old to signup'),
        MaxValueValidator(30, error_message='You are too old...'),
    ],
  )
  # One error message if any validator fails:
  age_2 = fields.IntegerField(
    validators=[MinValueValidator(18), MaxValueValidator(30)],
    error_message='Your age is invalid',
  )
```